### DISCUSSION TITLE
无法继续挑战了

### DISCUSSION THREAD

**Discussion by BirdSnail** [ByteLegend/ByteLegend#89] [General]
当我前进到`方法和字段`这块，就没有题目继续挑战了，是还没有开发完成吗？
![image](https://user-images.githubusercontent.com/39896910/140239702-7780d527-6a62-4c42-85be-5f26eb1c01de.png)

**Comment by gtn1024** [comment]
对的，等待更新

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: BirdSnail ***@***.***>
Sent: Thursday, November 4, 2021 8:59:15 AM
To: ByteLegend/ByteLegend ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ByteLegend/ByteLegend] 无法继续挑战了 (Discussion #89)


当我前进到方法和字段这块，就没有题目继续挑战了，是还没有开发完成吗？
[image]<https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F39896910%2F140239702-7780d527-6a62-4c42-85be-5f26eb1c01de.png&data=04%7C01%7C%7C20a9dfd769f54d21a74908d99f2e5333%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637715843580673044%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4MX%2B8kd0bHQzH7m0eLhk8dTKVeUJlkmOAuGMEA50lDk%3D&reserved=0>

―

**Comment by blindpirate** [comment]
正在做一个很大的新功能，所以游戏进度被耽搁了约两个星期。预计下周可以恢复进度。

**Reply by BirdSnail** [reply]
期待:smile:
